packages/@okta/vuepress-site/docs/guides/sign-in-to-spa-embedded-widget/main/index.md

```diff
@@ -14,7 +14,40 @@ title: Sign in to your SPA with the embedded Okta Sign-In Widget
 
 ## Create an Okta app integration
 
-<StackSnippet snippet="create-app-integration" />
+Before you integrate Okta authentication to your app, register your app in your Okta org. This provides you with the OpenID Connect client ID for authentication requests from your app. Register your app by creating an Okta app integration through the [Okta CLI](https://cli.okta.com/), the [Okta Apps API](/docs/reference/api/apps/), or through the [Admin Console](/docs/concepts/okta-organizations/#admin-console) with the following steps.
+
+1. In the Admin Console, go to **Applications** > **Applications**.
+1. Click **Create App Integration**.
+1. Select **OIDC - OpenID Connect** as the **Sign-in method**.
+1. Select **Single-Page Application** for the **Application Type**, and click **Next**.
+1. On the **New Single-Page App Integration** page:
+
+   * Enter an application name.
+   * Select the **Interaction Code** checkbox.
+   * Select the **Refresh Token** checkbox.
+   * Set **Sign-in redirect URIs** to a URI that is appropriate for your app. For example, `http://localhost:8080/login/callback` if you're using the sample app.
+   * Set **Sign-out redirect URIs** to a URI that is appropriate for your app. For example, `http://localhost:8080` if you're using the sample app.
+
+1. In the **Assignments** section, select **Allow everyone in your organization to access**, and click **Save**.
+1. In the **General Settings** section on the **General** tab, click **Edit**.
+1. Under **EMAIL VERIFICATION EXPERIENCE** set **Callback URI** to a URI that is appropriate for your app. For example, `http://localhost:8080/login` if you're using the sample app.
```

Comment:
Where did you get the `http://localhost:8080/login` value for angular (or is it all langs?)? I'm inclined to think it's '`http://localhost:8080/login/callback` since that what was used in NodeJS but I'm not totally sure. Let's talk about this in sync up. Shuo should know.

Reply:
/login/callback confirmed.  Updated